test/dlc_tests/ec2/pytorch/training/test_pytorch_training_2_2.py

```diff
@@ -34,6 +35,7 @@ def test_pytorch_2_2_gpu(
         (common_cases.nvapex, (pytorch_training, ec2_connection)),
         (common_cases.pytorch_training_torchaudio, (pytorch_training, ec2_connection)),
         (common_cases.pytorch_cudnn_match_gpu, (pytorch_training, ec2_connection, region)),
+        (common_cases.pytorch_curand_gpu, (pytorch_training, ec2_connection)),
```

Comment:
I checked `testCurand` file and [these lines](https://github.com/aws/deep-learning-containers/blob/3b23162c36308525c207edce1aff5925ce4c5a45/test/dlc_tests/container_tests/bin/testCurand#L9-L11) were changed  in PT 2.0 release to account for the fact that curand is shipped with conda as Lai mentioned.

I don't see a reason for removing this test since it is configured to test both pre and post conda distribution. But if you think it is an unnecessary overhead to maintain this test, then we can remove it.

Reply:
You can see the test time at the end of runs, I think it's only a couple seconds. If it can be run on multi-gpu instances without issue then I'm OK keeping it.

Reply:
Yeah, tests has been passing on multigpu instances. It only lasts 5 second so not much overhead.